spatialos-sdk/src/worker/internal/schema.rs

```diff
@@ -70,7 +70,20 @@ impl SchemaComponentUpdate {
         }
     }
 
-    // TODO: Cleared fields.
+    pub fn cleared_fields(&self) -> Vec<FieldId> {
```

Comment:
Would it be better to expose an API like `is_field_cleared`?

Reply:
Yes - you could store the cleared fields as a `Option<Vec<FieldId>>` when the object is created, then query it.

If this aligns with how the cleared fields API (I recall talking about the current implementation with you) - then this 100% makes sense